card/src/main/java/com/adyen/checkout/card/CardComponentState.java

```diff
@@ -20,6 +20,7 @@
 
     private final CardType mCardType;
     private final String mBinValue;
+    private final String mLastFourDigits;
```

Comment:
I wonder what this class would look like in Kotlin 👀

Reply:
Check it out!